README.md

```diff
@@ -322,9 +321,154 @@ func AsynchronousFriends(forUser: jimbob) -> Deferred<[Friend]> {
 }
 ```
 
-### Further Information
+## Mastering (the) `Future`
+
+Deferred is designed to scale with the fundamentals you see above. Large applications can be built using just `Deferred` and its `upon` and `fill` methods.
+
+### Read-Only Views
+
+It sometimes just doesn't make *sense* to be able to `fill` something; if you have a `Deferred` wrapping `UIApplication`'s push notification token, what does it mean if someone in your codebase calls `fill` on it?
+
+You may have noticed that anybody can call `upon` and `fill` on `Deferred` at any time; it's **read-write**. The former is fundamental, but the latter may be a liability as different pieces of code interact with each other. There's no concern to be had about thread safety, but instead about access: how do you limit illogical control flow, make it **read-only**?
```

Comment:
I don't get the "illogical control flow" bit. What control flow? I think that's unnecessary though. We can replace this whole sentence with just, "How can we make it read-only?"